stress/shell/src/main/java/alluxio/stress/cli/Benchmark.java

```diff
@@ -57,6 +57,13 @@
   @ParametersDelegate
   protected BaseParameters mBaseParameters = new BaseParameters();
 
+  /**
+   * Get the description of the bench. Shown with the usage of parameters.
```

Comment:
no need to say `Shown with the usage of parameters.`
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html
following oracle javadoc convention. method description is a contract between callers and implementations and needs to be implementation-independent. No need to say where it will be used, that's the duty of the callers. an exception will be saying where this method should not be used.